src/app.js

```diff
@@ -165,7 +165,20 @@ app.on('before-quit', () => {
 })
 
 function addCustomCss(windowElement) {
+  const cssBaseFile = 'style.css'
+
+  if (is.macos) {
```

Comment:
This condition is unfortunate.  It would be nice if there was an easier way to handle this, but I do not know of a way to do it currently.

Reply:
I don't like it either. "Is" doesn't have a generic verification method for putting it on a switch. Other solutions imho become a useless complication, perhaps more elegant